src/etc/inc/rrd.inc

```diff
@@ -795,7 +795,7 @@ function enable_rrd_graphing() {
 		/* End Captive Portal statistics */
 
 		/* NTP, set up the ntpd rrd file */
-		if (isset($config['ntpd']['statsgraph'])) {
+		if (isset($config['ntpd']['statsgraph']) && $config['ntpd']['enable'] != "off") {
```

Comment:
Because pretty much nothing is using boolean? Namely, the reason here was [this code](https://github.com/pfsense/pfsense/blob/master/src/etc/inc/service-utils.inc#L183) (though, after that I've decided to hide the thing from the status page as it seems it's the way it's being done with the rest of "built-in" services. Also see [#5510](https://redmine.pfsense.org/issues/5510)

So yeah, it's be nice to get some consistency here across the code, however this is not the place to do so and there's virtually no consent on what should be used.